ios/FluentUI/Core/CalendarConfiguration.swift

```diff
@@ -24,8 +22,11 @@ open class CalendarConfiguration: NSObject {
 
     @objc open var firstWeekday: Int = Calendar.current.firstWeekday
 
-    let referenceStartDate: Date
-    let referenceEndDate: Date
+    /// By default, this is today minus 3 years. If overridden, make sure it's before (less than) the 'referenceEndDate'
+    @objc open var referenceStartDate: Date
+
```

Comment:
Could we add a guard in didset, or not because it had @objc ?

Reply:
we could, but that would be assume clients will always set the start date first and then the end date... not sure if that's a correct assumption